.env.example

```diff
@@ -1,4 +1,2 @@
-REACT_APP_ARCHILOGIC_PUBLISHABLE_API_KEY=
-REACT_APP_API_BASE_URL=http://localhost:3000
-SERVER_ARCHILOGIC_SECRET_API_KEY=
-SERVER_ARCHILOGIC_API_BASE_URL=https://api.archilogic.com
\ No newline at end of file
+REACT_APP_ARCHILOGIC_PUBLISHABLE_API_KEY=4c847870-e59f-4d21-8210-b2c2d77f09ef
+REACT_APP_API_BASE_URL=https://api.archilogic.com
```

Comment:
Just to keep consistency across demos lets use the same variable name for archilogic api.
The one we are using in the room booking demo is: REACT_APP_ARCHILOGIC_API_URL